src/NzbDrone.Core/DecisionEngine/Specifications/RssSync/DelaySpecification.cs

```diff
@@ -45,6 +45,8 @@ public virtual DownloadSpecDecision IsSatisfiedBy(RemoteEpisode subject, SearchC
                 return DownloadSpecDecision.Accept();
             }
 
+            _logger.Debug("QualityProfile requires a waiting period of {0} minutes for {1}", delay, subject.Release.DownloadProtocol);
```

Comment:
`Delay Profile`, same for the other incorrect `QualityProfile` in the log (pretty sure it's a find/replace mistake when `Profile` was renamed back to `QualityProfile`